source/org.tensorflow/tensorflow-lite-support-api/Transforms/Metadata.xml

```diff
@@ -123,4 +115,29 @@
         Java.Lang.Object
     </attr>
 
+    <!-- 
+    cb_build
+
+    <attr
+        path="/api/package[@name='org.tensorflow.lite.support.common']/class[@name='TensorProcessor.Builder']/method[@name='build' and count(parameter)=0 and @return='org.tensorflow.lite.support.common.TensorProcessor']"
+        name="managedName"
+        >
+        BuildTensorProcessor
```

Comment:
Why doesn't setting `managedName` work?  What error are you trying to work around by manually defining `BuildTensorProcessor()`?